scapy/arch/common.py

```diff
@@ -43,8 +43,7 @@ def _check_tcpdump():
 
     # On some systems, --version does not exist on tcpdump
     return proc.returncode == 0 \
-        or output.startswith(b'Usage: tcpdump ') \
-        or output.startswith(b'tcpdump: unrecognized option')
+        or 'Usage: tcpdump ' in output
```

Comment:
```suggestion
        or b'Usage: tcpdump ' in output
```
Or did I miss something?

Reply:
Thanks. I pushed the modified files.